--- md5sums	2018-03-03 03:58:26.000000000 +0800
+++ md5sums-working	2018-03-13 17:33:10.551133903 +0800
@@ -1,5 +1,5 @@
-a61415312426e9c2212bd7dc7929abda  acl-2.2.52.src.tar.gz
-84f58dec00b60f2dc8fd1c9709291cc7  attr-2.4.47.src.tar.gz
+15ee11211b371081fdf3d25cf76d7b9c  acl-2.2.52.tar.gz
+22ca62bb22813eade2c6c4fcab246ec4  attr-2.4.47.tar.gz
 50f97f4159805e374639a73e2636f22e  autoconf-2.69.tar.xz
 24cd3501b6ad8cd4d7e2546f07e8b4d4  automake-1.15.1.tar.xz
 518e2c187cc11a17040f0915dddce54e  bash-4.4.18.tar.gz
@@ -39,12 +39,12 @@
 6a39bccf420c946b0fd7ffc64961315b  less-530.tar.gz
 6666b839e5d46c2ad33fc8aa2ceb5f77  libcap-2.25.tar.xz
 83b89587607e3eb65c70d361f13bab43  libffi-3.2.1.tar.gz
-b7437a5020190cfa84f09c412db38902  libpipeline-1.5.0.tar.gz
+2178a2fc66d95e73ad87b6126eeb47ee  libpipeline-1.5.0.tar.gz
 1bfb9b923f2c1339b4d2ce1807064aa5  libtool-2.4.6.tar.xz
 c74d30ec13491aeb24c237d703eace3e  linux-4.15.3.tar.xz
 730bb15d96fffe47e148d1e09235af82  m4-1.4.18.tar.xz
 15b012617e7c44c0ed482721629577ac  make-4.2.1.tar.bz2
-51842978e06686286421f9498d1009b7  man-db-2.8.1.tar.xz
+865d83a06ad138e424568c9b12ae62c2  man-db-2.8.1.tar.gz
 4298feb3d5feffad8ff46bb87b061a07  man-pages-4.15.tar.xz
 26a7ca93ec9cea5facb365664261f9c6  meson-0.44.0.tar.gz
 4125404e41e482ec68282a2e687f6c73  mpc-1.1.0.tar.gz
